explainer.md

```diff
@@ -257,41 +216,12 @@ Thus, a fallback option for platforms that do not support arbitrary characters
 
 ## FAQ
 
-### What data types are supported in different operating systems?
-
-See above.
-
-### Could the API take a fallback type?
-
-The proposal is: what if instead of just taking *one of* the string or integer,
-we allow sites to pass both, with an order of preference. This way, you could
-supply a string, but if strings aren't supported, fall back to a given number,
-or vice versa.
-
-This is something we're considering. My concern is that it makes the API too
-complex, where practically it isn't required (e.g., I believe all major
-platforms support at least one character strings).
-
-### Why limit the support to a single grapheme cluster? Is there a technical limitation?
```

Comment:
Could you bring this back and rephrase it as "Why limit the support to just an integer? What about other characters?"

And explain that we might consider adding support for other characters or symbols in the future.